src/routes/events.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+
+const router = express.Router();
+
+const eventsController = require('../controllers/events');
+
+router.get('/:id', eventsController.getOneEvent);
```

Comment:
You are not validating the id parameter. Someone can pass anything here and when you try